guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -44,6 +41,24 @@ $ cd foreman-discovery-image
 ----
 
 .Procedure
+. Open the `foreman-discovery-image.ks` file for editing:
++
+[options="nowrap" subs="+quotes"]
+----
+# vim 00-repos-centos8.ks
+----
```

Comment:
> Is this correct? I am asking because I see a different file name in the instruction than in the terminal example below it.

It's based on the old instructions (line 129 in the new version). The heading is indeed a copy-paste error.



> I also wonder if we should instruct user to explicitly use vim (which can be confusing if the user never encountered it before) instead of suggesting to simply replace the lines in the file and leaving the method to the user.

Good question. Perhaps that's better.